src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj

```diff
@@ -10,14 +10,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <RuntimeDependency Include="runtime.win7-amd64.runtime.native.System.Data.SqlClient.sni">
-      <TargetRuntime>win7-amd64</TargetRuntime>
+    <Dependency Include="runtime.win7-amd64.runtime.native.System.Data.SqlClient.sni">
       <Version>4.0.1-$(ExternalExpectedPrerelease)</Version>
-    </RuntimeDependency>
-    <RuntimeDependency Include="runtime.win7-x86.runtime.native.System.Data.SqlClient.sni">
-      <TargetRuntime>win7-x86</TargetRuntime>
+    </Dependency>
+    <Dependency Include="runtime.win7-x86.runtime.native.System.Data.SqlClient.sni">
       <Version>4.0.1-$(ExternalExpectedPrerelease)</Version>
-    </RuntimeDependency>
+    </Dependency>
```

Comment:
Per @ericstj 
"These have to be Dependencies rather than RuntimeDependencies. SqlClient has to be a fat package since it isn't in the shared framework, fat packages cannot rely on runtime.json."